Scripts/Runtime/Entities/TaskSystem/TaskFlow/TaskFlowSystem.cs

```diff
@@ -19,11 +19,18 @@ public TaskFlowSystem()
             TaskFlowGraph = new TaskFlowGraph();
         }
 
-        protected override void OnUpdate()
+        protected override void OnStartRunning()
         {
+            base.OnStartRunning();
+            
+            TaskFlowGraph.ConfigureTaskSystemJobs();
```

Comment:
is it ok that this will be executed more than once per run session? It will be executed anytime the system moves from disabled to enabled.

Obviously this won't happen during normal execution but what if the system is enabled/disabled in the editor?

Reply:
Good point, added a gate